$OpenBSD: patch-src_qalc_cc,v 1.2 2014/11/30 05:41:34 brad Exp $

Unbreak build (wrong header path, triggered by SEPARATE_BUILD).

Replace the malloc.h header with stdlib.h.

--- src/qalc.cc.orig	Tue Jan  5 10:17:29 2010
+++ src/qalc.cc	Thu Nov 20 00:26:42 2014
@@ -9,14 +9,14 @@
     (at your option) any later version.
 */
 
-#include "support.h"
+#include <libqalculate/support.h>
 #include <libqalculate/qalculate.h>
 #include <sys/stat.h>
 #include <unistd.h>
 #include <time.h>
 #include <pthread.h>
 #include <dirent.h>
-#include <malloc.h>
+#include <stdlib.h>
 #include <stdio.h>
 #include <vector>
 #include <glib.h>
